text/2021-05-10-substitute-rocksdb-write-stall.md

```diff
@@ -0,0 +1,115 @@
+# Substitute rocksdb write stall
+
```

Comment:
```suggestion

- RFC PR: https://github.com/tikv/rfcs/pull/67
- Tracking Issue: https://github.com/tikv/tikv/issues/10137

```

Reply:
I have done it for you as well as fixing the lint

PTAL https://github.com/Connor1996/rfcs/pull/1

Reply:
Thanks!